pwndbg/gdblib/events.py

```diff
@@ -12,8 +12,7 @@
 
 import pwndbg.config
 
-debug = pwndbg.config.Parameter("debug-events", False, "display internal event debugging info")
-pause = 0
+debug = pwndbg.config.Parameter("debug-events", True, "display internal event debugging info")
```

Comment:
You probably don't want to make the default `True`?

Reply:
No, good catch; that was from other testing